frontend/src/features/admin-form/create/payment/PaymentContent.tsx

```diff
@@ -67,7 +67,11 @@ export const PaymentContent = (): JSX.Element => {
                       <Text textStyle="body-1" textColor="secondary.700">
                         Your credit card will be charged:{' '}
                         <Text as="span" fontWeight="bold">
-                          S${((payments?.amount_cents ?? 0) / 100).toFixed(2)}
+                          S$
+                          {(payments?.payment_amount === undefined
+                            ? 0
+                            : Number(payments.payment_amount)
+                          ).toFixed(2)}
```

Comment:
/nit falsy check is probably good enough

```suggestion
                          {(payments?.payment_amount
                            ? 0
                            : Number(payments.payment_amount)
                          ).toFixed(2)}
```